python/ray/util/collective/requirements.txt

```diff
@@ -1 +1 @@
-cupy-cuda100
\ No newline at end of file
+cupy-cuda12x
```

Comment:
Hmm it is kind of backward incompatible changes though I don't think anyone uses our collective apis at this point. should we create a separate requirement.txt for adag?

Reply:
Removed this change.